packages/deployed-backend-client/src/deployed_backend_client_factory.ts

```diff
@@ -121,7 +121,7 @@ export class DeployedBackendClientFactory {
   /**
    * Returns a single instance of DeploymentClient
    */
-  static getInstance(
+  getInstance(
```

Comment:
we need to communicate this change to the console team since this will be a breaking change that impacts them.

Taking a step back, is this change worth it? Maybe we just enable this lint rule with ignore comments on existing instances and moving forward we adhere to the rule.

Reply:
if we ship 0.minor increment then per resolution rules they won't absorb update automagically and when they do their action is simple. 

as long as we follow semver I don't think we should block this change due to small cost on single consumer side.

Reply:
(if we were GA'd I'd recommend otherwise).

Reply:
Done and revert 😂

Reply:
Heads up for this breaking change @johnpc